### PR TITLE
fix: make configurable Tx lifetime

### PR DESCRIPTION
**Pull request by riemannulus**
- make configurable lifetime on `VolatileStagePolicy<T>`

### CODE REVIEW

**Review by longfin — APPROVED** [review]

**Review by dahlia — APPROVED** [review]
It seems to make `txLifeTime` configurable, rather than opting out it?

**Comment by riemannulus** [comment]
> It seems to make `txLifeTime` configurable, rather than opting out it?

You're right 😅 I'll amend this commit message to fix incorrect commit message and `txLifetime` typo.

**Comment by longfin** [comment]
It would be better if we mention `txLifeTime` on `README.md`.

**Comment by riemannulus** [comment]
> It would be better if we mention `txLifeTime` on `README.md`.

https://github.com/planetarium/NineChronicles.Headless/pull/258/commits/62945bf0a9d72c7ff985e12e07c15060ab715877 add it

**Review by longfin — APPROVED** [review]

**Review by dahlia — APPROVED** [review]

File: NineChronicles.Headless.Executable/Program.cs [hunk 1]
```diff
@@ -125,7 +125,9 @@ static void ConfigureSentryOptions(SentryOptions o)
             [Option(Description = "The AWS region for AWS CloudWatch (e.g., us-east-1, ap-northeast-2).")]
             string awsRegion = null,
             [Option(Description = "Run as an authorized miner, which mines only blocks that should be authorized.")]
-            bool authorizedMiner = false
+            bool authorizedMiner = false,
+            [Option(Description = "The lifetime of each Tx, It's unit is minute.  60 (m) by default.")]
```
Comment (moreal) [inline NineChronicles.Headless.Executable/Program.cs:129]: What you intended is below code, doesn't it?

```suggestion
            [Option(Description = "The lifetime of each Tx, its unit is minute.  60 (m) by default.")]
```

And how about using the full word? (e.g. `tx` and `transaction`)

```suggestion
            [Option(Description = "The lifetime of each transaction, which uses minute as its unit.  60 (m) by default.")]
```

Comment (riemannulus) [reply NineChronicles.Headless.Executable/Program.cs]: fix on https://github.com/planetarium/NineChronicles.Headless/pull/258/commits/ded53676b59171f558faf35a8d7d1b340eee3df4

**Review by moreal — APPROVED** [review]
👍

**Pull request merged**